objects/official/station/openrct2.station.noplatformnoentrance.json

```diff
@@ -12,7 +12,8 @@
     "strings": {
         "name": {
             "en-GB": "None, and no platform",
-            "nl-NL": "Geen, ook geen perron"
+            "nl-NL": "Geen, ook geen perron",
+            "ko-KR": "감추기, 승강장도 감추기",
```

Comment:
JSON unfortunately does not allow trailing commas, so you'll have to remove it from this line.

Reply:
Oops, sorry 👍